app/services/timer-service.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+  _countdown() {
+    if (this.get('seconds') === 0) {
+      this.set('timeUp', true);
+    } else {
+      this.incrementProperty('seconds', -1);
+      this._timer = Ember.run.later(this, '_countdown', 1000);
```

Comment:
This recursive function never has an end state. I suppose if timeUp is true then it won't display negative seconds, but seconds keeps counting down forever anyway?